shared/constants/fs.js

```diff
@@ -251,13 +251,17 @@ const itemStylesKeybase = {
   textType: folderTextType,
 }
 
-const getIconSpecFromUsernames = (usernames: Array<string>, me?: ?string) => {
-  if (usernames.length === 1) {
-    return makeAvatarPathItemIconSpec(usernames[0])
-  } else if (usernames.length > 1) {
-    return makeAvatarsPathItemIconSpec(usernames.filter(username => username !== me))
-  }
-  return makeBasicPathItemIconSpec('iconfont-question-mark', unknownTextColor)
+const getIconSpecFromUsernames = (usernames?: ?Array<string>, me?: ?string) => {
```

Comment:
does it really have to have so many optional params?

Reply:
I'll pass in an empty array instead.